src/Microsoft.TemplateEngine.Orchestrator.RunnableProjects/RunnableProjectGenerator.cs

```diff
@@ -118,58 +119,41 @@ public IParameterSet GetParametersForTemplate(IEngineEnvironmentSettings environ
+            foreach (IFile file in folder.EnumerateFiles(TemplateConfigFileName, SearchOption.AllDirectories))
             {
-                if (string.Equals(file.Name, TemplateConfigFileName, StringComparison.OrdinalIgnoreCase))
-                {
-                    IFile hostConfigFile = file.MountPoint.EnvironmentSettings.SettingsLoader.FindBestHostTemplateConfigFile(file);
-
-                    if (TryGetTemplateFromConfigInfo(file, out ITemplate template, hostTemplateConfigFile: hostConfigFile))
-                    {
-                        templateList.Add(template);
-                    }
-
-                    continue;
-                }
+                IFile hostConfigFile = file.MountPoint.EnvironmentSettings.SettingsLoader.FindBestHostTemplateConfigFile(file);
 
-                Match localeMatch = localeFileRegex.Match(file.Name);
-                if (localeMatch.Success)
+                if (TryGetTemplateFromConfigInfo(file, out ITemplate template, hostTemplateConfigFile: hostConfigFile))
                 {
-                    string locale = localeMatch.Groups["locale"].Value;
+                    templateList.Add(template);
 
-                    if (TryGetLangPackFromFile(file, out ILocalizationModel locModel))
+                    IDirectory localizeFolder = file.Parent.DirectoryInfo("localize");
+                    if (localizeFolder != null && localizeFolder.Exists)
                     {
-                        ILocalizationLocator locator = new LocalizationLocator()
+                        foreach (IFile locFile in localizeFolder.EnumerateFiles("*" + LocalizationFileSuffix, SearchOption.AllDirectories))
                         {
-                            Locale = locale,
-                            MountPointUri = source.MountPointUri,
-                            ConfigPlace = file.FullPath,
-                            Identity = locModel.Identity,
-                            Author = locModel.Author,
-                            Name = locModel.Name,
-                            Description = locModel.Description,
-                            ParameterSymbols = locModel.ParameterSymbols
-                        };
-                        localizations.Add(locator);
+                            string locale = locFile.Name.Substring(0, locFile.Name.Length - LocalizationFileSuffix.Length);
+
+                            if (TryGetLangPackFromFile(locFile, out ILocalizationModel locModel))
+                            {
+                                ILocalizationLocator locator = new LocalizationLocator()
+                                {
+                                    Locale = locale,
+                                    MountPointUri = source.MountPointUri,
+                                    ConfigPlace = locFile.FullPath,
+                                    Identity = locModel.Identity,
```

Comment:
Since code later uses this to match... At least for now, you want to use `template.Identity` here...

Reply:
I searched the repo and that Identity is never used anywhere as far as I can see. I plan to remove it if that is really the case.

Reply:
Here is used https://github.com/dotnet/templating/blob/main/src/Microsoft.TemplateEngine.Edge/Settings/TemplateCache.cs#L376 and later its used at https://github.com/dotnet/templating/blob/main/src/Microsoft.TemplateEngine.Edge/Settings/TemplateCache.cs#L204 to actually match it with "parent" template.

Reply:
That is `ILocalizationLocator.Identity`, I will not change it. I meant `ILocalizationModel.Identity` isn't used.